doc/conf.py

```diff
@@ -21,11 +21,11 @@
 # -- Project information -----------------------------------------------------
 
 project = "Spicy"
-copyright = "2023 by the Zeek Project"
+copyright = "2024 by the Zeek Project"
 author = "Zeek Project"
 
 version = open("../VERSION").readline()
-release = "1.9.0"  # most recent release version
+release = "1.11.0"  # most recent release version
```

Comment:
This one is still in development,

```suggestion
release = "1.10.0"  # most recent release version
```